index.js

```diff
@@ -25,7 +25,7 @@ module.exports = (api, options) => {
 
     webpackConfig.output.devtoolNamespace(name);
 
-    webpackConfig.set("devtool", "sourcemap");
+    webpackConfig.set("devtool", "source-map");
```

Comment:
This change appears to be compatible with both webpack 4 and webpack 5 👍🏿 

https://v4.webpack.js.org/configuration/devtool/#devtool